content/blog/2022-06-29-Galaxy-Wellbeing-Project/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Galaxy Well-being and Mental Health Project" 
+authors: "Melkeberhan Berhanu Degefa, Beatriz Serrano-Solano, Anne Fouilloux"
+date: "2022-06-29"
+---
+
+| ![Watercolor Painting](content/blog/2022-06-29-Galaxy-Wellbeing-Project/Wellbeing Project Image.jpg) |
```

Comment:
the filename with spaces is the only thing that looks concerning to me :) The rest looks great!

Reply:
ohh i see what you mean. I can change the name and upload it again and use that